### DISCUSSION TITLE
Test not running !!!

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2639] [Q&A] [answered]
![Screenshot (51)](https://github.com/user-attachments/assets/5dd6b16c-d456-4369-8202-d1c152a53401)

**Comment by Shiva-Sai-ssb** [comment, marked as answer]
Save the file. @prakharmishra3956

**Reply by prakharmishra3956** [reply]
thanks its done.
